.travis.yml

```diff
@@ -4,41 +4,34 @@ language:
   - node_js
 
 node_js:
-  - 8
+  - '10'
 
 services:
   - docker
 
-branches:
-  except:
-    - /^v\d+\.\d+\.\d+(-rc\.\d+)?$/
+if: tag IS blank # do not build tags
 
-notifications:
-  slack: smartcar:PGJmjtKFPytU5E2FwsImdVp8
+before_install:
+  - docker run --name selenium-container -d --net host -v /dev/shm:/dev/shm selenium/standalone-chrome
 
 install:
-  - npm install
-
-script:
-  - docker run --name selenium-container -d --net host -v /dev/shm:/dev/shm selenium/standalone-chrome
-  - npm run lint
-  - npm test
-  - npm run cover
-  - npm run build
-
-  - git config --global user.email "ci@smartcar.com"
-  - git config --global user.name "Travis CI User"
-  - export tag=$(grep 'version' package.json | sed 's/[", ]//g' | cut -d ':' -f 2)
-  - if [ "$TRAVIS_BRANCH" = "master" ]; then git tag -a v$tag -m "Travis Generated Tag"; fi
-
-deploy:
-  - provider: script
-    skip_cleanup: true
-    script: npm run publish
-    on:
-      branch: master
-  - provider: script
-    skip_cleanup: true
-    script: echo -e "machine github.com\n  login $CI_USER_TOKEN" >> ~/.netrc && git push origin v$tag
-    on:
-      branch: master
+  - npm ci
+
+stages:
+  - test
+  - publish
+
+jobs:
+  include:
+    - stage: test
+      node_js: '10'
+      script:
+        - npm test
+        - npm run cover
+          # need a local clone to let semantic-release git push --dry-run
+        - git clone --bare https://github.com/smartcar/javascript-sdk.git
+        - npx semantic-release --branch "$TRAVIS_BRANCH" --extends ./build/sr-configs/verify.js
+    - stage: publish
+      node_js: '10'
+      if: branch = master
```

Comment:
is this different than:
```yaml
on:
    branch: master
```

Reply:
no sir, this is the new conditonal syntax https://docs.travis-ci.com/user/conditional-builds-stages-jobs/